pyro/contrib/oed/eig.py

```diff
@@ -147,6 +147,69 @@ def naive_rainforth_eig(model, design, observation_labels, target_labels=None,
     return (conditional_lp - marginal_lp).sum(0)/N
 
 
+def accelerated_rainforth_eig(model, design, observation_labels, target_labels,
+                              yspace, N=100, M_prime=None):
+    """
+    Accelerated Rainforth (i.e. Unnested Monte Carlo) estimate of the expected information
+    gain (EIG). The estimate is
+
+    .. math::
+TO DO ################################################
+        \\frac{1}{N}\\sum_{n=1}^N \\log p(y_n | \\theta_n, d) -
+        \\log \\left(\\frac{1}{M}\\sum_{m=1}^M p(y_n | \\theta_m, d)\\right)
```

Comment:
Update this formula to respect the current implementations. Use two formulae for the two versions (with and without random effects). Could you also update the docstring for naive Rainforth to include the random effects formula? Once you have done all the doc changes, build the documentation and check it renders correctly.